src/docfx/Errors.cs

```diff
@@ -127,6 +127,9 @@ public static Error ExceedGitHubRateLimit()
         public static Error GitHubUserNotFound()
             => new Error(ErrorLevel.Warning, "github-user-not-found", $"User not found on GitHub");
 
+        public static Error GitHubCommitNotFound()
+            => new Error(ErrorLevel.Warning, "github-commit-not-found", $"Commit not found on GitHub");
```

Comment:
>GitHub [](start = 94, length = 6)

better to print out commit id

Reply:
added and changed this to an exception, with outer code to wrap it to a user friedly warning message.

---
In reply to: [209419977](https://github.com/dotnet/docfx/pull/3176#discussion_r209419977) [](ancestors = 209419977)